Basic Rules.txt

```diff
@@ -927,149 +1063,193 @@ three flavors: light, medium, and heavy; these indicate how wide of an area
 is suppressed as well as how effective that suppression is (defined below).
 The type of suppression available to you is dependent on your weapon.
 
-When you Suppress, you aim at the target as part of the action and fire a
+	When you Suppress, you aim at the target as part of the action and fire a
 hail of bullets (within your weapon range). Suppression doesn't require any
 rolls to initiate, the suppressor simply fires, creating a light/medium/heavy
 suppression AOE (Area-of-Effect, typically a 1-3m circle). Even though Suppression
 is an AOE effect, if you cross in-front of someone (someone who is suppressing 
 a different person) you will likely trigger automatic attack rolls, DM's discretion.
 
-If a suppressed target moves out into the open**, they trigger automatic attack
-rolls (or automatic hits) by the suppressor, once per target per suppressor's 
-turn. If the suppressed target shoots, they fire with +1 miss chance, not-stackable.
-Suppression status can't be purged unless you move outside the AOE. 
-The only other way to stop being suppressed is to stop the suppressor from 
-using a suppression action on you at the end of their turn.
+	If a suppressed target moves out into the open**, they trigger automatic 
+attack rolls (or automatic hits) by the suppressor, once per target per 
+suppressor's turn. If the suppressed target shoots, they fire with +1 miss 
+chance, not-stackable. Suppression status can't be purged unless you move 
+outside the AOE. The only other way to stop being suppressed is to stop the 
+suppressor from using a suppression action on you at the end of their turn.
+
+Moving out into the open: 
+	In the context of suppression and movement, "moving out into the open" 
+includes moving into a new location while you can be hit by shots from the 
+suppressor (this excludes moving into complete concealment, such as going prone 
+behind a high concrete wall). For example, the following situations would 
+trigger a free hit by the suppressor and give you the 'suppressed' status while 
+remaining in the area (let's assume you are upright (not crouched or prone)):
+moving from partial cover to an adjacent partial cover, moving from complete 
+concealment behind a barricade to partial cover, and moving when you are caught 
+out in the open.

```

Comment:
I think this topic should remain as a footnote, it will probably be need to be referenced several times beyond the initial understanding of Suppression. Otherwise, the couple of "**" -referenced elsewhere in the Suppression text- need to be removed.

Reply:
It's a multi-paragraph footnote...

Reply:
I think we were talking about that on the design guide, but we never resolved it? For reference, here is a multi paragraph footnote in literature:

![image](https://user-images.githubusercontent.com/2192156/33860953-0259077c-de90-11e7-8032-cebdd4052ec3.png)

I don't know why you don't like them so much (why don't you like them?). I think they are very useful.

Reply:
We could combine it into one paragraph if that is the main hangup.

Reply:
Why does it need to be a footnote? It looks like it's working as just a section. How about just replacing the ** in line 1073 with "(see below)"?